### PR TITLE
fix/TR-731 remove explicitly height for media-player

### PR DESCRIPTION
**Pull request by meefox**
Related to https://oat-sa.atlassian.net/browse/TR-731
To test:
create a test using an item from the ticket (do not modify the item).

see media player should have auto height
Screenshot 2021-03-10 at 10 47 41

### CODE REVIEW

**Review by btamas — COMMENTED** [review]
I see your modification solves the issue, but this is not a fix, it is a breaking change. If you would do it only for client, I would accept as a fix, but this will effect all environments and all clients. I'm smelling a regression issue here.

**Comment by meefox** [comment]
> I see your modification solves the issue, but this is not a fix, it is a breaking change. If you would do it only for nfer, I would accept as a fix, but this will effect all environments and all clients. I'm smelling a regression issue here.

Thank you, good point I will double-check if this NEFR only needed fix.

**Review by jsconan — COMMENTED** [review]
Caution!!! This is a harmful change that will introduce a possible regression.
Please check this PR: https://github.com/oat-sa/tao-core-ui-fe/pull/243

File: src/mediaplayer.js [hunk 1]
```diff
@@ -1944,6 +1951,30 @@ const mediaplayer = {
...
+    },
+
+    /**
+     * Set max height limit for container
+     * using by old media items with defined height.
+     * @private
+     */
+    _setMaxHeight: function _setMaxHeight() {
+        const $player = this.$container.find('.player');
+        const $video = this.$container.find('video.video', $player);
```
Comment (btamas) [inline src/mediaplayer.js:1967]: What is the second parameter of find? Based on the documentation (https://api.jquery.com/find/) is has only one parameter.

Comment (meefox) [reply src/mediaplayer.js]: My bad I confused with the general lookup selector where is the second parameter is a context.

File: src/mediaplayer.js [hunk 2]
```diff
@@ -1944,6 +1951,30 @@ const mediaplayer = {
...
+        }
+    },
+
+    /**
+     * Set max height limit for container
+     * using by old media items with defined height.
+     * @private
+     */
+    _setMaxHeight: function _setMaxHeight() {
+        const $video = this.$container.find('video.video');
```
Comment (bziondik) [inline src/mediaplayer.js:1966]: This check should be only for video (not youtube). And use `videoWidth` and `videoHeight` see example: 
https://github.com/oat-sa/tao-core-ui-fe/blob/4c85f2a4416124ea82d310ff01b4697f5a47b77b/src/mediaplayer.js#L1448-L1460

**Review by bziondik — CHANGES_REQUESTED** [review]

**Review by btamas — APPROVED** [review]
It works correctly with the test item.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by jsconan — APPROVED** [review]
Working well with the test item. As well as with created items

Unfortunately, the unit tests are failing.
You can try to run it in your browser:
`npm run test:keepAlive`
then open `/test/mediaplayer/test.html` on the launched server

However, I approve the PR since the tests are also failing in the `develop` branch.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

File: src/mediaplayer.js (hunk 2 above)
Comment (meefox) [reply src/mediaplayer.js:1966]: Thanks for the review.
1) why not youtube? (but it is skipping all new items with height: 'auto')
2) this is different, maybe I should rename properties. Here I using the box sizes of the video container.
Basically, I scale down the width of the container to make sure it's fitting to the max height.
Without that width will be pushed to 100%.

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: src/mediaplayer.js (hunk 2 above)
Comment (bziondik) [reply src/mediaplayer.js:1966]: Thanks!

**Pull request merged**